src/components/network-modification-tree-pane.js

```diff
@@ -182,6 +182,31 @@ export const NetworkModificationTreePane = ({
         [studyUuid, dispatch]
     );
 
+    const isSubtreeImpacted = useCallback(
+        (headerName) => {
+            const nodesImpacted = Array.isArray(
```

Comment:
Depending on the notification header containing node information we have to format the variable to test its value, its either an array or a single value

Reply:
Maybe `isSubtreeImpacted` should accept an array of nodes as parameter no matters what.
It should be independant of the notification specific state `studyUpdatedForce.eventData.headers` I think. 

Then when consuming notification you pass either `isSubtreeImpacted([studyUpdatedForce.eventData.headers['parentNode']])` or
`isSubtreeImpacted(studyUpdatedForce.eventData.headers['nodes'])` 
(see brackets)